modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -83,115 +91,141 @@ def _read(cls, filepath_or_buffer, **kwargs):
                 index_col = list(range(num_cols - len(names)))
                 if len(index_col) == 1:
                     index_col = index_col[0]
-                kwargs["index_col"] = index_col
```

Comment:
if you delete this line, then why create an empty dataframe on the previous lines?

Reply:
`index_col` still will be in the `partitions_kwargs` (`index_col` added to `partiotion_kwargs` [here](https://github.com/amyskov/modin/blob/text_io_refactoring/modin/engines/base/io/text/csv_dispatcher.py#L112)). This line was removed to avoid changing original `kwargs`.